File: Season 2013/Bogota/Projects/06.Agile_Business/backend/app/models/task.py
import logging
from google.appengine.ext import ndb
from endpoints_proto_datastore.ndb.model import EndpointsModel, EndpointsAliasProperty
from app.models.goal import GoalModel

class TaskModel(EndpointsModel):
	goal = ndb.KeyProperty(GoalModel)
	title = ndb.StringProperty()
	description = ndb.StringProperty()
	state = ndb.IntegerProperty()
	taskType = ndb.IntegerProperty()
	user = ndb.KeyProperty(UserModel)
	completed = ndb.BooleanProperty()

File: Season 2013/Bogota/Projects/06.Agile_Business/frontend/js/model/task.js
var User = function () {
	var self = this;

	this.goal = new Goal();
	this.title = ko.observable("");
	this.desciption = ko.observable("");
	this.state = ko.observable(0);
	this.taskType = ko.observable(0);
	this.user = new User();
	this.completed = ko.observable(false);
}